example/integration_test/all_test.dart

```diff
@@ -51,6 +51,16 @@ import 'viewport_test.dart' as viewport_test;
 import 'interactive_features_test.dart' as interactive_features_test;
 
 void main() {
+  // annotation tests
+  circle_annotation_manager_test.main();
+  circle_annotation_test.main();
+  point_annotation_manager_test.main();
+  point_annotation_test.main();
+  polygon_annotation_manager_test.main();
+  polygon_annotation_test.main();
+  polyline_annotation_manager_test.main();
+  polyline_annotation_test.main();
+
```

Comment:
I moved these tests up here since I'm fixing the suffix 0 for mocking event channel